styles.css

```diff
@@ -538,4 +620,338 @@
+
+  .works {
+    width: 78%;
+  }
+
+  .btn-1,
+  .btn-11 {
+    width: 120px;
+    border: 1px solid #dfe1e6;
+    margin-top: 10px;
+    margin-left: 20px;
+  }
+
+  .text-border {
+    margin-top: 35px;
+    display: flex;
+    flex-direction: column;
+    align-items: start;
+  }
+
+  .card-content {
+    flex-direction: row;
+    padding: 15px;
+    height: 100%;
+  }
+
+  .extra-wh-bg {
+    height: 550px;
+    border: 1px solid white;
+  }
+
+  .mobile-card-title {
+    display: none;
+  }
+
+  .card-title-multi {
+    display: block;
+    color: #172b4d;
+    font-size: 32px;
+    font-weight: 700;
+    margin-bottom: 15px;
+    margin-top: 0;
+    margin-left: 20px;
+  }
+
+  .img-class-3 {
+    display: block;
+    width: 100%;
+    height: 99%;
+  }
+
+  .img-div-3 {
+    width: 100%;
+    margin-bottom: 0;
+    margin-right: 7px;
+    height: 100%;
+  }
+
+  .canopy-2-big {
+    display: block;
+    font-size: 16px;
+    font-weight: normal;
+    margin-bottom: 25px;
+    margin-left: 20px;
+  }
+
+  .canopy-2 {
+    display: none;
+  }
+
+  .card-text-2 {
+    display: block;
+    margin-left: 20px;
+    padding-right: 45px;
+    font-size: 16px;
+    word-spacing: 4px;
+    line-height: 28px;
+  }
+
+  .card-text-1,
+  .btn-hide {
+    display: none;
+  }
+
+  .css-list-big {
+    display: inline-block;
+  }
+
+  .btns-3,
+  .btns-33 {
+    margin-left: 20px;
+    margin-top: 20px;
+    margin-bottom: 40px;
+  }
+
+  .btns-33 {
+    margin-left: 0;
+  }
+
+  .btn-11 {
+    margin-left: 0;
+  }
+
+  .img-hide,
+  .title-hide,
+  .canopy-show,
+  .text-hide {
+    display: none;
+  }
+
+  .img-big {
+    display: block;
+    width: 600px;
+    height: 500px;
+    margin-right: 40px;
+    margin-top: 7px;
+    margin-left: 7px;
+  }
+
+  .title-show,
+  .canopy-hide,
+  .text-show,
+  .btn-show {
+    display: block;
+  }
+
+  .btn1-invert {
+    background-color: #4053fc;
+    color: #fff;
+    margin-left: 0;
+  }
+
+  .btn1-invert:hover {
+    background-color: #fff;
+    color: #4053fc;
+  }
+
+  .btn1-invert:visited {
+    background-color: #e5e5e5;
+    color: #5e6c84;
+  }
+
+  .hand {
+    display: block;
+    position: relative;
+    bottom: 40px;
+    right: -430px;
+  }
+
+  .third-hide,
+  .about-hide {
+    display: none;
+  }
+
+  .third-show {
+    display: block;
+  }
+
+  .about-show {
+    display: flex;
+    margin-top: 40px;
+    padding: 130px;
+    background-color: #fff;
+    border-top-right-radius: 80px;
+  }
+
+  .left {
+    flex-basis: 50%;
+  }
+
+  .right {
+    flex-basis: 50%;
+  }
+
+  .right ul {
+    display: flex;
+    flex-direction: column;
+    list-style-type: none;
+    width: 100%;
+  }
+
+  .right ul li {
+    width: 100%;
+  }
+
+  #about3 {
+    color: #091e42;
+    font-size: 40px;
+    margin-bottom: 10px;
+  }
+
+  .about4 {
+    color: #344563;
+    font-size: 17px;
+    margin-bottom: 10px;
+    padding-right: 70px;
+    margin-right: 30px;
+    word-spacing: 6px;
+    width: 87%;
```

Comment:
- [ ] Again, let's try as much as possible to stick to the [Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=gsOQH8yjQCeXfLIW-0). Therefore I suggest that you modify the padding-right and margin-right of the `.about4` and may be the padding of `.about-show` to ensure that the description is intact ☺️ .

Your design 😎 
![section](https://user-images.githubusercontent.com/99394374/220840041-a77ef806-5134-407c-a50b-29b6d823a385.png)

Figma design 😎 
![image](https://user-images.githubusercontent.com/99394374/220840124-f10d22e5-1b83-4480-93b0-3bd3e4cb0caa.png)